input/exercises/solutions/cs/test_basic_2018a.md

```diff
@@ -0,0 +1,50 @@
+title: Test základních znalostí 2018a
+--- 
+# Zatmění čtvercového měsíce
+ ``` matlab
+	  A=imread('kytka256.jpg');
```

Comment:
Zbytečná mezera.. Koneckonců všechny kódu mají zbytečnou mezeru. Přisuň to uplně doleva.